public/css/custom.css

```diff
@@ -0,0 +1,7 @@
+.alert-danger {
+  color: white !important;
```

Comment:
If you find that you need `!important`, something has gone wrong. You probably need to go and [relax the specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) of your CSS elsewhere.

If you make the text color `!important`, you should probably make the `background-color` `!important`, too.